parsers/UA.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import arrow
+import dateutil
+import requests
+
+MAP_GENERATION = {
+  'aes': 'nuclear',
+  'tec': 'other', # fossil 
+  'tes': 'coal',
+  'chpp': 'gas',
+  'hpp': 'hydro',
+  'vde': 'wind',
+  'gesgaes' : 'gesgaes', # hydro pumped storage
+  'consumptiongaespump' :'consumptiongaespump' #hydro run of river and poundage

```

Comment:
Here's how it should work:
- run of the river hydro should be under production > hydro, and values should be positive
- pumped storage should be under storage > hydro, and positive values should represent storage and negative values should represent generation.